### DISCUSSION TITLE
adaptive speed mod on regular multiplayer

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#17268] [Q&A] [answered]
for regular multiplayer, does the speed of the mod change based on all of the player's mistimings or does it just desync?

**Comment by smoogipoo** [comment]
It shouldn't be allowed for multiplayer for the time being, since we have no way to deal with players playing with different rates.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/17269
